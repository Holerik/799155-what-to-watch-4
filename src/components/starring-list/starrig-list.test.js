// starrig-list.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import StarringList from './starring-list.jsx';

const starring = [`Jessica Chastain`, `Idris Elba`, `Kevin Costner`];

it(`StarringList should render correctly`, () => {
  const tree = renderer
    .create(
        <StarringList
          stars={starring}
        />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
