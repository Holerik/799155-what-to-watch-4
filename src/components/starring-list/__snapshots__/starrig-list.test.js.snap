// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StarringList should render correctly 1`] = `
Array [
  "Jessica Chastain",
  " ",
  <br />,
  "Idris Elba",
  " ",
  <br />,
  "Kevin Costner",
  " ",
  <br />,
]
`;
